website/docs/components/form/masked-input/partials/guidelines/guidelines.md

```diff
@@ -84,3 +84,7 @@ Setting the `hasCopyButton` property to `true` will display a [Copy Button](/com
 ## Error validation
 
 For error validation recommendations, refer to the [Form patterns](/patterns/form-patterns) documentation.
+
+## Content
+
+For content recommendations and detailed guidance on using the character count feature in this field, please refer to our [Primitives](/components/form/primitives) documentation.
```

Comment:
For all of this related comments where the primitives are referenced in other component guidelines, how about this: Changing the content headline to "Content and Character count" (or something similar) so that the character count shows up in the sidecar, but doesn't result in a potential duplication of linking in subsequent paragraphs/sections? Something like:

## Content and Character count

For recommendations...blah blah blah rest of the statement pointing to the primitives broadely.

_Or_

Do you see value in breaking them up and linking directly to the Character count headline in the primitives?

Reply:
Personally, I see breaking them up as more accurate (character count is a feature, but content isn't) and scalable because additional guidance could always be added under either heading later.

Reply:
Broke all of these apart in https://github.com/hashicorp/design-system/pull/1799/commits/e82f392450e6bf962653dadbc63c74771984b881